src/main/java/uk/gov/justice/digital/delius/transformers/OffenderManagerTransformer.java

```diff
@@ -111,4 +113,13 @@ private static boolean isUnallocated(final PrisonOffenderManager offenderManager
                 .orElse(false);
     }
 
+    private static OffenderManagerGrade offenderManagerGradeOf(final OffenderManager offenderManager) {
+        return Optional.ofNullable(offenderManager.getOfficer().getGrade())
+            .map(grade -> OffenderManagerGrade.builder()
+                .code(grade.getCodeValue())
+                .description(grade.getCodeDescription())
+                .build())
+            .orElse(null);
```

Comment:
I do not really understand the use of orEsle(null) , that I can see is used everywhere , personally I believe is a bad practice and should be replaced with Optional.orElse()